src/ert/dark_storage/endpoints/records.py

```diff
@@ -141,22 +140,22 @@ def get_ensemble_responses(
     return response_map
 
 
-@router.get(
-    "/ensembles/{ensemble_id}/records/{key}/std_dev", response_model=js.ImageOut
-)
+@router.get("/ensembles/{ensemble_id}/records/{key}/std_dev")
 def get_std_dev(
     *, storage: Storage = DEFAULT_STORAGE, ensemble_id: UUID, key: str, z: int
-) -> js.ImageOut:
```

Comment:
Can the js.ImageOut be deleted with this change?

Reply:
Yes, nice catch.